exchanges.yml

```diff
@@ -217,17 +217,57 @@ components:
                     type: string
                   template:
                       type: string
-            workflowId:
+            id:
               type: string
-              description: the ID that will be used for the created workflow. workflowId is OPTIONAL.
+              description: The ID that will be used for the created workflow. Passing an ID is OPTIONAL.
     CreateWorkflowResponse:
       type: object
       additionalProperties: false
-      description: Object containing information about a created workflow.
+      description: Response containing the created workflowData Object.
```

Comment:
Change `workflowData` -> `config`.

Same changes with `workflowX` -> `X` (in general).